onnx/backend/test/cpp/driver/test_driver.cc

```diff
@@ -0,0 +1,187 @@
+#include "test_driver.h"
+#include <cstdio>
+#include <cstdlib>
+#include <fstream>
+
+#include <sys/stat.h>
+#ifdef _WIN32
+#include <windows.h>
+#else
+#include <dirent.h>
+#include <errno.h>
+#endif
+
+namespace onnx {
+namespace testing {
+bool FileExists(const std::string& filename) {
+  struct stat stats;
+  if (lstat(filename.c_str(), &stats) != 0 || !S_ISREG(stats.st_mode)) {
+    return false;
+  }
+  return true;
+}
+
+void TestDriver::SetDefaultDir(const std::string& s) {
```

Comment:
is this planned to be used, do not see any references.

Reply:
It is not yet to be used (including default_dir it self), but I think it would be useful in the future, and unharm to keep it now.